test/vm.py

```diff
@@ -107,18 +125,52 @@ def start(self):
             "-monitor", "none",
             "-netdev", f"user,id=net.0,hostfwd=tcp::{self._ssh_port}-:22",
             "-device", "rtl8139,netdev=net.0",
+            "-qmp", f"unix:{self._qmp_socket},server,nowait",
         ]
-        if self._snapshot:
+        if use_ovmf:
+            qemu_cmdline.extend(["-bios", find_ovmf()])
+        if self._cdrom:
+            qemu_cmdline.extend(["-cdrom", self._cdrom])
+        if snapshot:
             qemu_cmdline.append("-snapshot")
         qemu_cmdline.append(self._img)
         self._log(f"vm starting, log available at {log_path}")
 
         # XXX: use systemd-run to ensure cleanup?
         self._qemu_p = subprocess.Popen(
-            qemu_cmdline, stdout=sys.stdout, stderr=sys.stderr)
+            qemu_cmdline,
+            stdout=sys.stdout,
+            stderr=sys.stderr,
+        )
         # XXX: also check that qemu is working and did not crash
-        self.wait_ssh_ready()
-        self._log(f"vm ready at port {self._ssh_port}")
+        match wait_event.split(":"):
```

Comment:
Do we need to run these tests downstream as well at some point? The `match`-statement was added to a quite recent Python.

Reply:
As a general note, I would answer - yes.

Reply:
Thank you, indeed `match` is only available in python 3.10 and centos9  has 3.9 and centos8 has 3.6 so I remove this.

Reply:
This is done now